launch_helm.sh

```diff
@@ -0,0 +1,121 @@
+      echo AWS_MAIL_ACCOUNT_ACCESS_KEY_ID="$aws_mail_account_access_key_id"
+      echo AWS_MAIL_ACCOUNT_SECRET_ACCESS_KEY="$aws_mail_account_secret_access_key"
+      exit 1
+else
+      # Authentication with email
+      single_admin_email="$SINGLE_ADMIN_EMAIL"
+      aws_mail_account_access_key_id="$AWS_MAIL_ACCOUNT_ACCESS_KEY_ID"
+      aws_mail_account_secret_access_key="$AWS_MAIL_ACCOUNT_SECRET_ACCESS_KEY"
+
+      echo SINGLE_ADMIN_EMAIL="$single_admin_email"
+      echo AWS_MAIL_ACCOUNT_ACCESS_KEY_ID="$aws_mail_account_access_key_id"
+      echo AWS_MAIL_ACCOUNT_SECRET_ACCESS_KEY="$aws_mail_account_secret_access_key"
+fi
+
+
+helm install "$release_name-redis" bitnami/redis --set auth.enabled=false --namespace "$namespace" --create-namespace 1> /dev/null
+helm install "$release_name-postgres" bitnami/postgresql --set auth.username="$db_user" --set auth.password="$db_password" --namespace "$namespace" --create-namespace 1> /dev/null
+
+postgres_release="$release_name-postgres"
+postgres_host="$postgres_release-postgresql.$namespace.svc.cluster.local"
+
+echo postgres_release="$postgres_release"
+echo postgres_host="$postgres_host"
+
+redis_release="$release_name-redis"
+redis_host="$redis_release-master.$namespace.svc.cluster.local"
+
+
+wait_time=60
+echo "Waiting for ${wait_time} seconds for the pods to be up before continuing..."
+sleep ${wait_time}
+
+# This is needed as passing --set auth.database=$db_name does not work. The database does not get created.
+echo "Create the database with 'create database ${db_name};' in the following psql prompt. Exit when done."
+
+kubectl run "$postgres_release-postgresql-client" --rm --tty -i --restart='Never' --namespace "$namespace" --image docker.io/bitnami/postgresql:14.5.0-debian-11-r6 --env="PGPASSWORD=$db_password" \
+      --command -- psql --host "$postgres_host" -U "$db_user" -d postgres -p 5432
+
+if [ "$use_email_auth" = "false" ]; then
+      helm install --debug "$release_name" ./helm-chart --namespace "$namespace" --create-namespace \
+      --set dockerPullSecret="$docker_pull_secret" \
+      --set avatarVersion="$avatar_version" \
+      --set dbPassword="$db_password" \
+      --set dbName="$db_name" \
+      --set dbUser="$db_user" \
+      --set dbHost="$postgres_host" \
+      --set redisHost="$redis_host" \
+      --set api.pepper=$(python -c "import secrets; print(secrets.token_hex(), end='')") \
+      --set api.authjwtSecretKey=$(python -c "import secrets; print(secrets.token_hex(), end='')") \
+      --set api.fileEncryptionKey=$(python -c "import base64; import os; print(str(base64.urlsafe_b64encode(os.urandom(32)), encoding='utf-8'), end='')") \
+      --set api.avatarVersion="$avatar_version" \
+      --set api.useEmailAuthentication="$use_email_auth" \
+      --set api.firstUserName="$first_user_name" \
+      --set api.firstUserPassword="$first_user_password"
```

Comment:
Wouldn't it be clearer to refactor to only put the 2-3 different arguments in the if ?
(Looks very hard to maintain as is)